2024-06-03.mdx

```diff
@@ -0,0 +1,55 @@
+---
+date: 6-3-2024
+---
+
+### What's new with Xata
+
+**UI improvements**
+
+You might have noticed already, but we've been beautifying the UI in the past weeks. For example, check out this empty state page for a new database: 🤩
+
+![Empty state for the new database page](/images/2024-06-03-empty-state.png)
```

Comment:
We showcased this in the last changelog (3rd section down) https://github.com/xataio/mdx-changelog/blob/main/2024-05-22.mdx

I think the running multiple statements at once is actually a pretty big improvement in the playground UX -- our second most popular app behind the table. What do you think about a quick screen studio recording and leading with that? Happy to do it if you'd like.

Reply:
I did this. The video is not working in preview, but it should work once merged.